hips/tiles/surveys.py

```diff
@@ -14,11 +14,11 @@ class HipsSurveyPropertiesList:
 
     Parameters
     ----------
-    surveys : `~astropy.table.Table`
+    surveys : list
         HiPS surveys
     """
 
-    def __init__(self, surveys: Table) -> None:
+    def __init__(self, surveys: list) -> None:
```

Comment:
I think  for type annotations to be useful, you'd have to use `from typing import List` and declare the type as `List[HipsSurveyProperties]` here. I've never used that, but maybe you can try?